src/kvstore/RocksEngine.cpp

```diff
@@ -96,7 +96,8 @@ class RocksWriteBatch : public WriteBatch {
 RocksEngine::RocksEngine(GraphSpaceID spaceId,
                          const std::string& dataPath,
                          std::shared_ptr<rocksdb::MergeOperator> mergeOp,
-                         std::shared_ptr<rocksdb::CompactionFilterFactory> cfFactory)
+                         std::shared_ptr<rocksdb::CompactionFilterFactory> cfFactory,
+                         bool bOpenForWrite)
```

Comment:
seems we don't need to add a prefix `b` for naming variable

Reply:
good point, in vesoft style, no need.